src/web/views/index.js

```diff
@@ -89,7 +89,8 @@ module.exports = ({ config, metadata, url }) => {
           <script src="/js/config.js" defer="defer"></script>
           <script src="/js/vendor.bundle${BUNDLE_PREFIX}.js" defer="defer"></script>
           <script src="https://js.pusher.com/4.4/pusher.min.js"></script>
-          <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono" rel="stylesheet" type="text/css">
+          <link rel="preconnect" href="https://rsms.me/">
+          <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
```

Comment:
pulling the font from the font author's website.
Rationale:
- its backed by cloudflare so should have similar reliability as google fonts
- the google fonts version of this font has some bugs and is notorious for not being updated, so avoiding that